src/main/java/svc/data/citations/datasources/tyler/TylerViolationStatus.java

```diff
@@ -0,0 +1,37 @@
+package svc.data.citations.datasources.tyler;
+
+import svc.models.VIOLATION_STATUS;
+
+public enum TylerViolationStatus {
+
+	WARRANT_ISSUED("WARRANT ISSUED", VIOLATION_STATUS.FTA_WARRANT_ISSUED);
+
+	private final String tylerViolationStatusString;
+	private final VIOLATION_STATUS violationStatus;
+
+	private TylerViolationStatus(String tylerViolationStatusString, VIOLATION_STATUS violationStatus) {
+		this.tylerViolationStatusString = tylerViolationStatusString;
+		this.violationStatus = violationStatus;
+	}
+
+	public String getTylerStatusString() {
+		return this.tylerViolationStatusString;
+	}
+
+	public VIOLATION_STATUS getViolationStatus() {
+		return this.violationStatus;
+	}
+
+	public static TylerViolationStatus fromTylerViolationStatusString(String tylerViolationStatusString) {
```

Comment:
Sure. One is a pre-condition check (return out early) and the other two could be replaced with a variable and single return for sure.